### PR TITLE
fix(readme): update readme to match new guidelines

### PR DESCRIPTION
**Pull request by dgutride**
**What**: Closes https://github.com/patternfly/patternfly-react/issues/3753 
Updates the parent readme for the project with new content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,91 @@
...
+
+[PatternFly React seed](https://github.com/patternfly/patternfly-react-seed) is an open source build scaffolding utility. It gives developers a jump-start when creating new projects that will use PatternFly.
+
+
+1. **Fork the project**
+
+  * TypeScript: [Fork and clone the PatternFly React Seed project](https://github.com/patternfly/patternfly-react-seed).
+  * JavaScript: [Fork and clone the JavaScript branch of the PatternFly React Seed project](https://github.com/patternfly/patternfly-react-seed/tree/javascript).
+
+  For more information about how to fork and clone a repo, [view GitHub Help](https://help.github.com/articles/fork-a-repo/).
```
Comment (tlabaj) [inline README.md]: Should this be indented?  When I look at the rich text it is not aligned under the "1."  I don't know if it should be.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,91 @@
...
+
+2. **Install the project**
+
+  Follow the steps outlined in the [PatternFly React Seed README](https://github.com/patternfly/patternfly-react-seed#quick-start) to install.
+
```
Comment (tlabaj) [inline README.md]: Same question about alignment here.

**Comment by dgutride** [comment]
I added more spaces - see here:  https://github.com/dgutride/patternfly-react/tree/update-readme

File: README.md [hunk 3]
```diff
@@ -1,24 +1,91 @@
-# patternfly-react
+# PatternFly React
 
 [![lerna](https://img.shields.io/badge/maintained%20with-lerna-green.svg?style=for-the-badge)](https://lernajs.io/)
 [![PatternFly 3 npm badge](https://img.shields.io/npm/v/patternfly-react.svg?label=PF3%20Core&style=for-the-badge)](https://www.npmjs.com/package/patternfly-react)
 [![PatternFly 4 npm badge](https://img.shields.io/npm/v/@patternfly/react-core.svg?label=PF4%20Core&style=for-the-badge)](https://www.npmjs.com/package/@patternfly/react-core)
 [![CodeCov](https://img.shields.io/codecov/c/gh/patternfly/patternfly-react?style=for-the-badge)](https://codecov.io/gh/patternfly/patternfly-react)
 
-This project will provide a set of React UI components for the [PatternFly project](https://patternfly.org).
+This project provides a set of React components for the [PatternFly project](https://patternfly.org).
```
Comment (abigaeljamie) [inline README.md]:
```suggestion
This project provides a set of React components for PatternFly project](https://patternfly.org).  
```

File: README.md [hunk 4]
```diff
@@ -1,24 +1,91 @@
...
+Before you begin, check out this [overview of PatternFly](http://patternfly.org/v4/get-started/about) to get familiar with the basic elements of the design system.
+
+#### Install a package manager
+Install a package manager before using the PatternFly libraries.
+
+* [Use npm](https://nodejs.org/en/download)
+* [Use Yarn](https://yarnpkg.com/en/docs/getting-started)
+
+
+#### Start with PatternFly React Seed
```
Comment (abigaeljamie) [inline README.md]:
```suggestion
#### Start with PatternFly React seed
```

File: README.md (hunk 1 above)
Comment (dgutride) [reply README.md]: Updated alignment earlier.

File: README.md (hunk 2 above)
Comment (dgutride) [reply README.md]: Updated alignment earlier.

**Pull request merged**